asyncnsq/tcp/reader.py

```diff
@@ -197,7 +197,7 @@ async def _lookupd(self):
     async def unsubscribe(self):
         if not self._is_subscribe:
             raise ValueError('You must subscribe to the topic first')
-        logger.debug("unsubscribe starting... in {}".format(__package__))
+        # logger.debug("unsubscribing {}".format(self))
```

Comment:
debug message if only for you to debug you program, it is not a noise log message

Reply:
I added this message in debugging, and then removed it in the next commit to clean it up.